### ISSUE TITLE
数据集问题

### ISSUE THREAD

**Issue by zzhhzz666** [PKINet/PKINet#10]
### Prerequisite

- [X] I have searched [Issues](https://github.com/open-mmlab/mmrotate/issues) and [Discussions](https://github.com/open-mmlab/mmrotate/discussions) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://mmrotate.readthedocs.io/en/1.x/notes/4_faq.html) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version (master)](https://github.com/open-mmlab/mmrotate) or [latest version (1.x)](https://github.com/open-mmlab/mmrotate/tree/dev-1.x).

### Task

I'm using the official example scripts/configs for the officially supported tasks/models/datasets.

### Branch

master branch https://github.com/open-mmlab/mmrotate

### Environment

sys.platform: linux
Python: 3.7.13 (default, Mar 29 2022, 02:18:16) [GCC 7.5.0]
CUDA available: True
GPU 0,1,2,3: NVIDIA TITAN Xp
CUDA_HOME: /usr/local/cuda
NVCC: Cuda compilation tools, release 11.7, V11.7.64
GCC: gcc (Ubuntu 11.4.0-1ubuntu1~22.04) 11.4.0
PyTorch: 1.9.0
PyTorch compiling details: PyTorch built with:
  - GCC 7.3
  - C++ Version: 201402
  - Intel(R) Math Kernel Library Version 2020.0.2 Product Build 20200624 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v2.1.2 (Git Hash 98be7e8afa711dc9b66c8ff3504129cb82013cdb)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - NNPACK is enabled
  - CPU capability usage: AVX2
  - CUDA Runtime 11.1
  - NVCC architecture flags: -gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_61,code=sm_61;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_80,code=sm_80;-gencode;arch=compute_86,code=sm_86;-gencode;arch=compute_37,code=compute_37
  - CuDNN 8.0.5
  - Magma 2.5.2
  - Build settings: BLAS_INFO=mkl, BUILD_TYPE=Release, CUDA_VERSION=11.1, CUDNN_VERSION=8.0.5, CXX_COMPILER=/opt/rh/devtoolset-7/root/usr/bin/c++, CXX_FLAGS= -Wno-deprecated -fvisibility-inlines-hidden -DUSE_PTHREADPOOL -fopenmp -DNDEBUG -DUSE_KINETO -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DSYMBOLICATE_MOBILE_DEBUG_HANDLE -O2 -fPIC -Wno-narrowing -Wall -Wextra -Werror=return-type -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-sign-compare -Wno-unused-parameter -Wno-unused-variable -Wno-unused-function -Wno-unused-result -Wno-unused-local-typedefs -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-psabi -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Wno-stringop-overflow, LAPACK_INFO=mkl, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, TORCH_VERSION=1.9.0, USE_CUDA=ON, USE_CUDNN=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=ON, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON, 

TorchVision: 0.10.0
OpenCV: 4.10.0
MMCV: 1.7.2
MMCV Compiler: GCC 7.3
MMCV CUDA Compiler: 11.1
MMDetection: 2.13.0+
------------------------------------------------------------

### Reproduces the problem - code sample

无

### Reproduces the problem - command or script

无

### Reproduces the problem - error message

无

### Additional information

您好！我目前的baseline是使用的dior-r数据集，我想将这个工作替换我的backbone，但是条件比较差，磁盘没有空间了，目前无法下载Dotav1.0数据集。然后我看您提供的是dota数据集的训练权重和训练模型，我能直接换成dior吗，是需要自己先训练吗。
还有，我有点困惑的是，dota数据集不是已经分好训练集测试集了嘛，为啥还要先进行分割呢

**Comment by PKINet** [comment]
1. About DIOR-R training. 
You can change the base config in [pkinet-s_fpn_o-rcnn-dotav1-ss_le90.py]:(https://github.com/PKINet/PKINet/blob/main/configs/pkinet/pkinet-s_fpn_o-rcnn-dotav1-ss_le90.py)

`_base_ = [
    '../_base_/default_runtime.py', '../_base_/schedules/schedule_30e.py', '../_base_/datasets/dotav1_ss1024.py']`
into 
`_base_ = [
    '../_base_/default_runtime.py', '../_base_/schedules/schedule_30e.py', '../_base_/datasets/dior.py']`
 
change class number:
`num_classes = 15`
into
`num_classes = 20` 

and change resize scale:
`dict(type='RResize', img_scale=(1024, 1024))`
into
`dict(type='RResize', img_scale=(800, 800))`

After that, you can train and eval on the DIOR-R dataset.

2. About the images processing. 
DOTA dataset has training set, validation set, and test set. Following previous work, we train the network on the trainval joint dataset and inference on the test set and submit results to the official website for the fair comparison. Before training, we also need to split the images into a fixed size (e.g., 1024*1024) because remote sensing images sometimes have high resolutions and inconsistent scales.

**Comment by PKINet** [comment]
This issue is temporarily closed due to inactivity. Feel free to reopen it.
